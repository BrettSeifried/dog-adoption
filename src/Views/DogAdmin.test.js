import { render } from '@testing-library/react';
import DogCreate from './DogAdmin';
import { MemoryRouter } from 'react-router-dom';

test('renders admin page', () => {
  const { container } = render(
    <MemoryRouter>
      <DogCreate />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
