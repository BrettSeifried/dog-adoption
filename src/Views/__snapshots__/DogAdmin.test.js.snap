// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders admin page 1`] = `
<div>
  <div>
    <h1>
      Add a Dog
    </h1>
    <div
      class="form"
    >
      <form>
        <h3>
          Dog Edit Form
        </h3>
        <div
          class="dog-info"
        >
          <label>
            Name
          </label>
          <input
            placeholder="name"
            type="text"
            value=""
          />
        </div>
        <div
          class="dog-info"
        >
          <label>
            Breed
          </label>
          <input
            placeholder="Breed"
            type="text"
            value=""
          />
        </div>
        <div
          class="dog-info"
        >
          <label>
            Age
          </label>
          <input
            placeholder="Age"
            type="number"
            value=""
          />
        </div>
        <div
          class="dog-info"
        >
          <label>
            Bio
          </label>
          <input
            placeholder="Description of the Dog"
            type="text"
            value=""
          />
        </div>
        <div
          class="dog-info"
        >
          <label>
            Image URL
          </label>
          <input
            placeholder="Image URL Address"
            type="text"
            value=""
          />
        </div>
        <div>
          <button>
            Add/Edit
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
